<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="resTab">
      <attribute name="title">
       <string>Reservations</string>
      </attribute>
      <layout class="QGridLayout" name="resLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="countLabel">
         <property name="text">
          <string>Locked jobs count</string>
         </property>
         <property name="buddy">
          <cstring>countSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="countSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QScrollArea" name="typesScrool">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="typesWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>364</width>
            <height>175</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="typesLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="docTab">
      <attribute name="title">
       <string>Documents</string>
      </attribute>
      <layout class="QGridLayout" name="docScroolLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="docScrool">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="docWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>364</width>
            <height>206</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="docLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="jobTab">
      <attribute name="title">
       <string>Jobs</string>
      </attribute>
      <layout class="QGridLayout" name="jobScroolLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="jobScrool">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="jobWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>364</width>
            <height>206</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="jobLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>countSpin</tabstop>
  <tabstop>typesScrool</tabstop>
  <tabstop>docScrool</tabstop>
  <tabstop>jobScrool</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
